src/components/LeftPanel.tsx

```diff
@@ -28,7 +31,10 @@ const Wrapper = styled.div`
 export const LeftPanel = () => {
   return (
     <Wrapper>
-      <Div>left side</Div>
+      <Div>
+        <Twitter />
```

Comment:
You should add some space for the "bird" from above and the left, like on this photo

![image](https://user-images.githubusercontent.com/94183891/185244584-6f0768ff-e223-4d8b-82d1-3db97c3e84bd.png)